rules/typescript.js

```diff
@@ -1,11 +1,14 @@
 module.exports = {
   rules: {
     '@typescript-eslint/array-type': ['error', { default: 'array-simple' }],
+    '@typescript-eslint/ban-ts-comment': 'error',
     '@typescript-eslint/consistent-type-definitions': ['error', 'interface'],
+    '@typescript-eslint/comma-spacing': 'off',
```

Comment:
Let's add a comment why, you should be able to copy-paste one of the ~ "formatting should be done by Prettier".

Reply:
fixed with https://github.com/ridedott/eslint-config/pull/314/commits/8ad156eb1bfc14ba3f60a7cd101ef0207f4422f6